ci/ansible/roles/pulp-backup/defaults/main.yaml

```diff
@@ -0,0 +1,20 @@
+---
+# location of backup on remote
+fetch_to_localhost: true
+remote_bkup_dir: "{{ ansible_user_dir }}/remote_backups"
+local_bkup_dir/: "local_backups"
```

Comment:
Are you sure? I believe every usage of `local_bkup_dir` already inserts slashes as appropriate:

```console
$ git grep local_bkup_dir
pulp_backup.yaml:#                   -e local_bkup_dir=${HOME}/backups \
roles/pulp-backup/tasks/main.yaml:    dest: "{{ local_bkup_dir }}/{{ item.value.file }}.tar.bz2"
roles/pulp-backup/tasks/main.yaml:    dest: "{{ local_bkup_dir }}/{{ item.value.file }}.bz2"
roles/pulp-restore-backup/tasks/main.yaml:    src: "{{ local_bkup_dir }}/{{ item.value.archive }}"
roles/pulp-restore-backup/tasks/main.yaml:    src: "{{ local_bkup_dir }}/{{ item.value.archive }}"
[snip!]
```

I also ran this playbook locally without the slash and didn't encounter any issues. Then again, I made some other changes while fiddling around, too.